server/.env.example

```
@@ -13,11 +13,26 @@ JWT_SECRET=
 AWS_REGION=
 AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
+# See .env.example at the repository root for a Compose-oriented combined template.
+
+DATABASE_URL=postgresql://postgres:Secret@2811@localhost:5432/internhack?sslmode=disable
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The DATABASE_URL key is duplicated

(DuplicatedKey)